jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -251,16 +248,30 @@ public synchronized void closeTracer() {
   }
 
   /**
-   * @param statsFactory the factory
-   * @deprecated Use {@link #setStatsFactory(StatsFactory)} instead
+   * @see #setMetricsFactory(MetricsFactory)
+   * @param statsFactory the StatsFactory to use on the Tracer to be built
+   * @deprecated Use {@link #setMetricsFactory(MetricsFactory)} instead
    */
   @Deprecated
   public void setStatsFactor(StatsFactory statsFactory) {
-    this.statsFactory = statsFactory;
+    this.metricsFactory = statsFactory;
   }
 
+  /**
+   * @see #setMetricsFactory(MetricsFactory)
+   * @param statsFactory the StatsFactory to use on the Tracer to be built
+   * @deprecated Use {@link #setMetricsFactory(MetricsFactory)} instead
+   */
+  @Deprecated
   public void setStatsFactory(StatsFactory statsFactory) {
-    this.statsFactory = statsFactory;
+    this.metricsFactory = statsFactory;
+  }
+
+  /**
+   * @param metricsFactory the MetricsFactory to use on the Tracer to be built
+   */
+  public void setMetricsFactory(MetricsFactory metricsFactory) {
```

Comment:
nit: pending #346, the method name might change, so best to wait until we agree on the new naming pattern

Reply:
Other than that, is this PR good to be merged?